### PR TITLE
Library maintenance

### PR DESCRIPTION
**Pull request by tshemsedinov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,4 @@
-## sandboxedFs
-
-[![npm version](https://badge.fury.io/js/sandboxed-fs.svg)](https://badge.fury.io/js/sandboxed-fs)
-[![Build Status](https://travis-ci.org/metarhia/sandboxed-fs.svg?branch=master)](https://travis-ci.org/metarhia/sandboxed-fs)
-[![NPM Downloads/Month](https://img.shields.io/npm/dm/sandboxed-fs.svg)](https://www.npmjs.com/package/sandboxed-fs)
-[![NPM Downloads](https://img.shields.io/npm/dt/sandboxed-fs.svg)](https://www.npmjs.com/package/sandboxed-fs)
+## sandboxed-fs [![npm version](https://img.shields.io/npm/v/sandboxed-fs.svg?style=flat)](https://www.npmjs.com/package/sandboxed-fs) [![Build Status](https://travis-ci.org/metarhia/sandboxed-fs.svg?branch=master)](https://travis-ci.org/metarhia/sandboxed-fs) [![NPM Downloads/Month](https://img.shields.io/npm/dm/sandboxed-fs.svg)](https://www.npmjs.com/package/sandboxed-fs) [![NPM Downloads](https://img.shields.io/npm/dt/sandboxed-fs.svg)](https://www.npmjs.com/package/sandboxed-fs) [![license](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/metarhia/sandboxed-fs/blob/master/LICENSE)
```
Comment (belochub) [inline README.md]: Wrap the line please, there's no need to put all of the badges on the same line.

File: README.md [hunk 2]
```diff
@@ -1,9 +1,4 @@
...
 
 sandboxedFs is a sandboxed wrapper for Node.js file system module implementing
```
Comment (belochub) [inline README.md]: We need to change the name here as well.
```suggestion
sandboxed-fs is a sandboxed wrapper for Node.js file system module implementing
```

File: README.md (hunk 1 above)
Comment (tshemsedinov) [reply README.md]: If we will place all badges at the separate line we will see:
Screenshot from 2020-03-20 20-22-07
If we leave one badge at header line and all other at the separate:
Screenshot from 2020-03-20 20-23-10
Single long header line with all badges will look like:
Screenshot from 2020-03-20 20-23-56

File: README.md (hunk 2 above)
Comment (tshemsedinov) [reply README.md]: Fixed

File: README.md (hunk 1 above)
Comment (belochub) [reply README.md]: I don't see the need to put all of the badges in the header, but if it's required, it can be done with wrapping using the `<h1>` tags.

Comment (tshemsedinov) [reply README.md]: Ok, I changed all badges to html too because github doesn't support md inside html.

**Review by belochub — APPROVED** [review]
LGTM

**Comment by belochub** [comment]
Landed in https://github.com/metarhia/sandboxed-fs/compare/fbd7a7c4a215d9630cefa6800b139e88f8edd57f...f12ba0394191001bd908c353d69846678a04e8b5.

**Pull request closed without merging**